packages/components/src/types/generated/graphql.tsx

```diff
@@ -11137,13 +11135,23 @@ export type UnifiedSearchVenue = {
   name?: Maybe<LanguageString>;
   ontologyWords?: Maybe<Array<Maybe<OntologyWord>>>;
   openingHours?: Maybe<OpeningHours>;
+  orderedByAccessibilityShortcoming?: Maybe<AccessibilityShortcoming>;
   partOf?: Maybe<UnifiedSearchVenue>;
   reservationPolicy?: Maybe<VenueReservationPolicy>;
   resources: Array<Resource>;
   serviceOwner?: Maybe<ServiceOwner>;
   targetGroups?: Maybe<Array<Maybe<TargetGroup>>>;
 };
 
+/**
+ * A place that forms a unit and can be used for some specific purpose -
+ * respa unit or resource, service map unit, beta.kultus venue, linked
+ * events place, Kukkuu venue
+ */
```

Comment:
This comment is automatically copied by `yarn generate:graphql` from the comment of UnifiedSearchVenue type, see https://github.com/City-of-Helsinki/unified-search/blob/578d112907e4da1aaed0d6543dd031759a24b875/graphql/src/schemas/location.ts#L3-L8